2023-05-17 10:07:44,477 WARN (fe|54) [FrontendOptions.initAddrUseIp():243] ip address range configured for priority_networks does not include the current IP address
2023-05-17 10:07:44,711 WARN (fe|54) [StateChangeExecutor.notifyNewFETypeTransfer():62] notify new FE type transfer: LEADER
2023-05-17 10:07:44,998 WARN (stateChangeExecutor|61) [AuthenticationProviderFactory.installPlugin():32] Plugin MYSQL_NATIVE_PASSWORD has already been installed!
2023-05-17 10:07:44,999 WARN (stateChangeExecutor|61) [AuthenticationProviderFactory.installPlugin():32] Plugin AUTHENTICATION_LDAP_SIMPLE has already been installed!
2023-05-17 10:07:45,000 WARN (stateChangeExecutor|61) [AuthenticationProviderFactory.installPlugin():32] Plugin AUTHENTICATION_KERBEROS has already been installed!
2023-05-17 10:07:45,001 WARN (stateChangeExecutor|61) [AuthenticationProviderFactory.installPlugin():32] Plugin AUTHENTICATION_LDAP_SIMPLE_FOR_EXTERNAL has already been installed!
2023-05-17 10:07:45,404 WARN (Routine load task scheduler|48) [RoutineLoadTaskScheduler.process():117] no available be slot to scheduler tasks, wait for 10 seconds to scheduler again, you can set max_routine_load_task_num_per_be bigger in fe.conf, current value is 5
2023-05-17 10:07:53,081 WARN (agent-task-pool-2|103) [AgentBatchTask.run():205] task exec error. backend[10001]
org.apache.thrift.transport.TTransportException: backend 10001 shutdown
	at com.starrocks.pseudocluster.PseudoBackend$BeThriftClient.submit_tasks(PseudoBackend.java:735) ~[test-classes/:?]
	at com.starrocks.task.AgentBatchTask.run(AgentBatchTask.java:195) [classes/:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) [?:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) [?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
	at java.lang.Thread.run(Thread.java:829) [?:?]
2023-05-17 10:07:53,447 WARN (heartbeat-mgr-pool-6|100) [HeartbeatMgr$BackendHeartbeatHandler.call():336] backend heartbeat got exception, addr: 127.0.0.10:12101
org.apache.thrift.transport.TTransportException: backend 10001 shutdown
	at com.starrocks.pseudocluster.PseudoBackend$HeartBeatClient.heartbeat(PseudoBackend.java:707) ~[test-classes/:?]
	at com.starrocks.system.HeartbeatMgr$BackendHeartbeatHandler.call(HeartbeatMgr.java:296) [classes/:?]
	at com.starrocks.system.HeartbeatMgr$BackendHeartbeatHandler.call(HeartbeatMgr.java:274) [classes/:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) [?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
	at java.lang.Thread.run(Thread.java:829) [?:?]
2023-05-17 10:07:53,453 WARN (heartbeat mgr|40) [HeartbeatMgr.runAfterCatalogReady():177] get bad heartbeat response: type: BACKEND, status: BAD, msg: backend 10001 shutdown
2023-05-17 10:07:54,652 WARN (starrocks-mysql-nio-pool-1|98) [BackendServiceClient.execBatchPlanFragmentsAsync():131] Execute batch plan fragments catch a exception, address=127.0.0.10:12100
java.lang.RuntimeException: backend 10001 shutdown
	at com.starrocks.pseudocluster.PseudoBackend$PseudoPBackendService.execBatchPlanFragmentsAsync(PseudoBackend.java:896) ~[test-classes/:?]
	at com.starrocks.rpc.BackendServiceClient.execBatchPlanFragmentsAsync(BackendServiceClient.java:117) ~[classes/:?]
	at com.starrocks.qe.Coordinator$BackendExecState.execRemoteBatchFragmentsAsync(Coordinator.java:2065) ~[classes/:?]
	at com.starrocks.qe.Coordinator.deliverExecBatchFragmentsRequests(Coordinator.java:1071) ~[classes/:?]
	at com.starrocks.qe.Coordinator.deliverExecFragments(Coordinator.java:609) ~[classes/:?]
	at com.starrocks.qe.Coordinator.exec(Coordinator.java:537) ~[classes/:?]
	at com.starrocks.qe.StmtExecutor.handleDMLStmt(StmtExecutor.java:1459) ~[classes/:?]
	at com.starrocks.qe.StmtExecutor.execute(StmtExecutor.java:516) ~[classes/:?]
	at com.starrocks.qe.ConnectProcessor.handleQuery(ConnectProcessor.java:359) ~[classes/:?]
	at com.starrocks.qe.ConnectProcessor.dispatch(ConnectProcessor.java:473) ~[classes/:?]
	at com.starrocks.qe.ConnectProcessor.processOnce(ConnectProcessor.java:739) ~[classes/:?]
	at com.starrocks.mysql.nio.ReadListener.lambda$handleEvent$0(ReadListener.java:69) ~[classes/:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
	at java.lang.Thread.run(Thread.java:829) [?:?]
2023-05-17 10:07:54,662 WARN (starrocks-mysql-nio-pool-1|98) [Coordinator.deliverExecBatchFragmentsRequests():1103] exec plan fragment failed, errmsg=backend 10001 shutdown, host: 127.0.0.10, code: THRIFT_RPC_ERROR, fragmentId=F00, backend=127.0.0.10:12102
2023-05-17 10:07:54,672 WARN (starrocks-mysql-nio-pool-1|98) [BackendServiceClient.cancelPlanFragmentAsync():174] Cancel plan fragment catch a exception, address=127.0.0.10:12100
java.lang.RuntimeException: backend 10001 shutdown
	at com.starrocks.pseudocluster.PseudoBackend$PseudoPBackendService.cancelPlanFragmentAsync(PseudoBackend.java:938) ~[test-classes/:?]
	at com.starrocks.rpc.BackendServiceClient.cancelPlanFragmentAsync(BackendServiceClient.java:157) ~[classes/:?]
	at com.starrocks.qe.Coordinator$BackendExecState.cancelFragmentInstance(Coordinator.java:1973) ~[classes/:?]
	at com.starrocks.qe.Coordinator.cancelRemoteFragmentsAsync(Coordinator.java:1468) ~[classes/:?]
	at com.starrocks.qe.Coordinator.cancelInternal(Coordinator.java:1455) ~[classes/:?]
	at com.starrocks.qe.Coordinator.handleErrorBackendExecState(Coordinator.java:619) ~[classes/:?]
	at com.starrocks.qe.Coordinator.deliverExecBatchFragmentsRequests(Coordinator.java:1122) ~[classes/:?]
	at com.starrocks.qe.Coordinator.deliverExecFragments(Coordinator.java:609) ~[classes/:?]
	at com.starrocks.qe.Coordinator.exec(Coordinator.java:537) ~[classes/:?]
	at com.starrocks.qe.StmtExecutor.handleDMLStmt(StmtExecutor.java:1459) ~[classes/:?]
	at com.starrocks.qe.StmtExecutor.execute(StmtExecutor.java:516) ~[classes/:?]
	at com.starrocks.qe.ConnectProcessor.handleQuery(ConnectProcessor.java:359) ~[classes/:?]
	at com.starrocks.qe.ConnectProcessor.dispatch(ConnectProcessor.java:473) ~[classes/:?]
	at com.starrocks.qe.ConnectProcessor.processOnce(ConnectProcessor.java:739) ~[classes/:?]
	at com.starrocks.mysql.nio.ReadListener.lambda$handleEvent$0(ReadListener.java:69) ~[classes/:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
	at java.lang.Thread.run(Thread.java:829) [?:?]
2023-05-17 10:07:54,674 WARN (starrocks-mysql-nio-pool-1|98) [Coordinator$BackendExecState.cancelFragmentInstance():1976] cancel plan fragment get a exception, address=127.0.0.10:12100
2023-05-17 10:07:54,701 WARN (starrocks-mysql-nio-pool-1|98) [SimpleScheduler.addToBlacklist():171] add black list 10001
2023-05-17 10:07:54,702 WARN (starrocks-mysql-nio-pool-1|98) [StmtExecutor.handleDMLStmt():1620] failed to handle stmt [insert into test.test_1 values (1,"1", 1), (2,"2", 2), (3,"3", 3)] label: insert_a2a074b8-f457-11ed-b47c-0242ac110003
com.starrocks.rpc.RpcException: rpc failed, host: 127.0.0.10
	at com.starrocks.qe.Coordinator.handleErrorBackendExecState(Coordinator.java:625) ~[classes/:?]
	at com.starrocks.qe.Coordinator.deliverExecBatchFragmentsRequests(Coordinator.java:1122) ~[classes/:?]
	at com.starrocks.qe.Coordinator.deliverExecFragments(Coordinator.java:609) ~[classes/:?]
	at com.starrocks.qe.Coordinator.exec(Coordinator.java:537) ~[classes/:?]
	at com.starrocks.qe.StmtExecutor.handleDMLStmt(StmtExecutor.java:1459) ~[classes/:?]
	at com.starrocks.qe.StmtExecutor.execute(StmtExecutor.java:516) ~[classes/:?]
	at com.starrocks.qe.ConnectProcessor.handleQuery(ConnectProcessor.java:359) ~[classes/:?]
	at com.starrocks.qe.ConnectProcessor.dispatch(ConnectProcessor.java:473) ~[classes/:?]
	at com.starrocks.qe.ConnectProcessor.processOnce(ConnectProcessor.java:739) ~[classes/:?]
	at com.starrocks.mysql.nio.ReadListener.lambda$handleEvent$0(ReadListener.java:69) ~[classes/:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
	at java.lang.Thread.run(Thread.java:829) [?:?]
2023-05-17 10:07:54,713 WARN (starrocks-mysql-nio-pool-1|98) [StmtExecutor.execute():521] DML statement(insert into test.test_1 values (1,"1", 1), (2,"2", 2), (3,"3", 3)) process failed.
com.starrocks.common.UserException: rpc failed, host: 127.0.0.10
	at com.starrocks.qe.StmtExecutor.handleDMLStmt(StmtExecutor.java:1664) ~[classes/:?]
	at com.starrocks.qe.StmtExecutor.execute(StmtExecutor.java:516) ~[classes/:?]
	at com.starrocks.qe.ConnectProcessor.handleQuery(ConnectProcessor.java:359) ~[classes/:?]
	at com.starrocks.qe.ConnectProcessor.dispatch(ConnectProcessor.java:473) ~[classes/:?]
	at com.starrocks.qe.ConnectProcessor.processOnce(ConnectProcessor.java:739) ~[classes/:?]
	at com.starrocks.mysql.nio.ReadListener.lambda$handleEvent$0(ReadListener.java:69) ~[classes/:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
	at java.lang.Thread.run(Thread.java:829) [?:?]
2023-05-17 10:07:55,464 WARN (heartbeat-mgr-pool-4|96) [HeartbeatMgr$BackendHeartbeatHandler.call():336] backend heartbeat got exception, addr: 127.0.0.10:12101
org.apache.thrift.transport.TTransportException: backend 10001 shutdown
	at com.starrocks.pseudocluster.PseudoBackend$HeartBeatClient.heartbeat(PseudoBackend.java:707) ~[test-classes/:?]
	at com.starrocks.system.HeartbeatMgr$BackendHeartbeatHandler.call(HeartbeatMgr.java:296) [classes/:?]
	at com.starrocks.system.HeartbeatMgr$BackendHeartbeatHandler.call(HeartbeatMgr.java:274) [classes/:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) [?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
	at java.lang.Thread.run(Thread.java:829) [?:?]
2023-05-17 10:07:55,466 WARN (heartbeat mgr|40) [HeartbeatMgr.runAfterCatalogReady():177] get bad heartbeat response: type: BACKEND, status: BAD, msg: backend 10001 shutdown
2023-05-17 10:07:55,825 WARN (agent-task-pool-3|104) [AgentBatchTask.run():205] task exec error. backend[10001]
org.apache.thrift.transport.TTransportException: backend 10001 shutdown
	at com.starrocks.pseudocluster.PseudoBackend$BeThriftClient.submit_tasks(PseudoBackend.java:735) ~[test-classes/:?]
	at com.starrocks.task.AgentBatchTask.run(AgentBatchTask.java:195) [classes/:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) [?:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) [?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
	at java.lang.Thread.run(Thread.java:829) [?:?]
2023-05-17 10:07:57,357 WARN (agent-task-pool-3|104) [AgentBatchTask.run():205] task exec error. backend[10001]
org.apache.thrift.transport.TTransportException: backend 10001 shutdown
	at com.starrocks.pseudocluster.PseudoBackend$BeThriftClient.submit_tasks(PseudoBackend.java:735) ~[test-classes/:?]
	at com.starrocks.task.AgentBatchTask.run(AgentBatchTask.java:195) [classes/:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) [?:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) [?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
	at java.lang.Thread.run(Thread.java:829) [?:?]
2023-05-17 10:07:57,474 WARN (heartbeat-mgr-pool-5|99) [HeartbeatMgr$BackendHeartbeatHandler.call():336] backend heartbeat got exception, addr: 127.0.0.10:12101
org.apache.thrift.transport.TTransportException: backend 10001 shutdown
	at com.starrocks.pseudocluster.PseudoBackend$HeartBeatClient.heartbeat(PseudoBackend.java:707) ~[test-classes/:?]
	at com.starrocks.system.HeartbeatMgr$BackendHeartbeatHandler.call(HeartbeatMgr.java:296) [classes/:?]
	at com.starrocks.system.HeartbeatMgr$BackendHeartbeatHandler.call(HeartbeatMgr.java:274) [classes/:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) [?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
	at java.lang.Thread.run(Thread.java:829) [?:?]
2023-05-17 10:07:57,477 WARN (heartbeat mgr|40) [HeartbeatMgr.runAfterCatalogReady():177] get bad heartbeat response: type: BACKEND, status: BAD, msg: backend 10001 shutdown
2023-05-17 10:07:58,875 WARN (starrocks-mysql-nio-pool-1|98) [BackendServiceClient.execBatchPlanFragmentsAsync():131] Execute batch plan fragments catch a exception, address=127.0.0.10:12100
java.lang.RuntimeException: backend 10001 shutdown
	at com.starrocks.pseudocluster.PseudoBackend$PseudoPBackendService.execBatchPlanFragmentsAsync(PseudoBackend.java:896) ~[test-classes/:?]
	at com.starrocks.rpc.BackendServiceClient.execBatchPlanFragmentsAsync(BackendServiceClient.java:117) ~[classes/:?]
	at com.starrocks.qe.Coordinator$BackendExecState.execRemoteBatchFragmentsAsync(Coordinator.java:2065) ~[classes/:?]
	at com.starrocks.qe.Coordinator.deliverExecBatchFragmentsRequests(Coordinator.java:1071) ~[classes/:?]
	at com.starrocks.qe.Coordinator.deliverExecFragments(Coordinator.java:609) ~[classes/:?]
	at com.starrocks.qe.Coordinator.exec(Coordinator.java:537) ~[classes/:?]
	at com.starrocks.qe.StmtExecutor.handleDMLStmt(StmtExecutor.java:1459) ~[classes/:?]
	at com.starrocks.qe.StmtExecutor.execute(StmtExecutor.java:516) ~[classes/:?]
	at com.starrocks.qe.ConnectProcessor.handleQuery(ConnectProcessor.java:359) ~[classes/:?]
	at com.starrocks.qe.ConnectProcessor.dispatch(ConnectProcessor.java:473) ~[classes/:?]
	at com.starrocks.qe.ConnectProcessor.processOnce(ConnectProcessor.java:739) ~[classes/:?]
	at com.starrocks.mysql.nio.ReadListener.lambda$handleEvent$0(ReadListener.java:69) ~[classes/:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
	at java.lang.Thread.run(Thread.java:829) [?:?]
2023-05-17 10:07:58,877 WARN (starrocks-mysql-nio-pool-1|98) [Coordinator.deliverExecBatchFragmentsRequests():1103] exec plan fragment failed, errmsg=backend 10001 shutdown, host: 127.0.0.10, code: THRIFT_RPC_ERROR, fragmentId=F00, backend=127.0.0.10:12102
2023-05-17 10:07:58,879 WARN (starrocks-mysql-nio-pool-1|98) [BackendServiceClient.cancelPlanFragmentAsync():174] Cancel plan fragment catch a exception, address=127.0.0.10:12100
java.lang.RuntimeException: backend 10001 shutdown
	at com.starrocks.pseudocluster.PseudoBackend$PseudoPBackendService.cancelPlanFragmentAsync(PseudoBackend.java:938) ~[test-classes/:?]
	at com.starrocks.rpc.BackendServiceClient.cancelPlanFragmentAsync(BackendServiceClient.java:157) ~[classes/:?]
	at com.starrocks.qe.Coordinator$BackendExecState.cancelFragmentInstance(Coordinator.java:1973) ~[classes/:?]
	at com.starrocks.qe.Coordinator.cancelRemoteFragmentsAsync(Coordinator.java:1468) ~[classes/:?]
	at com.starrocks.qe.Coordinator.cancelInternal(Coordinator.java:1455) ~[classes/:?]
	at com.starrocks.qe.Coordinator.handleErrorBackendExecState(Coordinator.java:619) ~[classes/:?]
	at com.starrocks.qe.Coordinator.deliverExecBatchFragmentsRequests(Coordinator.java:1122) ~[classes/:?]
	at com.starrocks.qe.Coordinator.deliverExecFragments(Coordinator.java:609) ~[classes/:?]
	at com.starrocks.qe.Coordinator.exec(Coordinator.java:537) ~[classes/:?]
	at com.starrocks.qe.StmtExecutor.handleDMLStmt(StmtExecutor.java:1459) ~[classes/:?]
	at com.starrocks.qe.StmtExecutor.execute(StmtExecutor.java:516) ~[classes/:?]
	at com.starrocks.qe.ConnectProcessor.handleQuery(ConnectProcessor.java:359) ~[classes/:?]
	at com.starrocks.qe.ConnectProcessor.dispatch(ConnectProcessor.java:473) ~[classes/:?]
	at com.starrocks.qe.ConnectProcessor.processOnce(ConnectProcessor.java:739) ~[classes/:?]
	at com.starrocks.mysql.nio.ReadListener.lambda$handleEvent$0(ReadListener.java:69) ~[classes/:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
	at java.lang.Thread.run(Thread.java:829) [?:?]
2023-05-17 10:07:58,883 WARN (starrocks-mysql-nio-pool-1|98) [Coordinator$BackendExecState.cancelFragmentInstance():1976] cancel plan fragment get a exception, address=127.0.0.10:12100
2023-05-17 10:07:58,885 WARN (starrocks-mysql-nio-pool-1|98) [SimpleScheduler.addToBlacklist():171] add black list 10001
2023-05-17 10:07:58,886 WARN (starrocks-mysql-nio-pool-1|98) [StmtExecutor.handleDMLStmt():1620] failed to handle stmt [insert into test.test_1 values (1,"1", 1), (2,"2", 2), (3,"3", 3)] label: insert_a529909d-f457-11ed-b47c-0242ac110003
com.starrocks.rpc.RpcException: rpc failed, host: 127.0.0.10
	at com.starrocks.qe.Coordinator.handleErrorBackendExecState(Coordinator.java:625) ~[classes/:?]
	at com.starrocks.qe.Coordinator.deliverExecBatchFragmentsRequests(Coordinator.java:1122) ~[classes/:?]
	at com.starrocks.qe.Coordinator.deliverExecFragments(Coordinator.java:609) ~[classes/:?]
	at com.starrocks.qe.Coordinator.exec(Coordinator.java:537) ~[classes/:?]
	at com.starrocks.qe.StmtExecutor.handleDMLStmt(StmtExecutor.java:1459) ~[classes/:?]
	at com.starrocks.qe.StmtExecutor.execute(StmtExecutor.java:516) ~[classes/:?]
	at com.starrocks.qe.ConnectProcessor.handleQuery(ConnectProcessor.java:359) ~[classes/:?]
	at com.starrocks.qe.ConnectProcessor.dispatch(ConnectProcessor.java:473) ~[classes/:?]
	at com.starrocks.qe.ConnectProcessor.processOnce(ConnectProcessor.java:739) ~[classes/:?]
	at com.starrocks.mysql.nio.ReadListener.lambda$handleEvent$0(ReadListener.java:69) ~[classes/:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
	at java.lang.Thread.run(Thread.java:829) [?:?]
2023-05-17 10:07:58,891 WARN (starrocks-mysql-nio-pool-1|98) [StmtExecutor.execute():521] DML statement(insert into test.test_1 values (1,"1", 1), (2,"2", 2), (3,"3", 3)) process failed.
com.starrocks.common.UserException: rpc failed, host: 127.0.0.10
	at com.starrocks.qe.StmtExecutor.handleDMLStmt(StmtExecutor.java:1664) ~[classes/:?]
	at com.starrocks.qe.StmtExecutor.execute(StmtExecutor.java:516) ~[classes/:?]
	at com.starrocks.qe.ConnectProcessor.handleQuery(ConnectProcessor.java:359) ~[classes/:?]
	at com.starrocks.qe.ConnectProcessor.dispatch(ConnectProcessor.java:473) ~[classes/:?]
	at com.starrocks.qe.ConnectProcessor.processOnce(ConnectProcessor.java:739) ~[classes/:?]
	at com.starrocks.mysql.nio.ReadListener.lambda$handleEvent$0(ReadListener.java:69) ~[classes/:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
	at java.lang.Thread.run(Thread.java:829) [?:?]
2023-05-17 10:07:59,480 WARN (heartbeat-mgr-pool-1|93) [HeartbeatMgr$BackendHeartbeatHandler.call():336] backend heartbeat got exception, addr: 127.0.0.10:12101
org.apache.thrift.transport.TTransportException: backend 10001 shutdown
	at com.starrocks.pseudocluster.PseudoBackend$HeartBeatClient.heartbeat(PseudoBackend.java:707) ~[test-classes/:?]
	at com.starrocks.system.HeartbeatMgr$BackendHeartbeatHandler.call(HeartbeatMgr.java:296) [classes/:?]
	at com.starrocks.system.HeartbeatMgr$BackendHeartbeatHandler.call(HeartbeatMgr.java:274) [classes/:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) [?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
	at java.lang.Thread.run(Thread.java:829) [?:?]
2023-05-17 10:07:59,482 WARN (heartbeat mgr|40) [HeartbeatMgr.runAfterCatalogReady():177] get bad heartbeat response: type: BACKEND, status: BAD, msg: backend 10001 shutdown
2023-05-17 10:08:01,507 WARN (heartbeat-mgr-pool-0|77) [HeartbeatMgr$BackendHeartbeatHandler.call():336] backend heartbeat got exception, addr: 127.0.0.10:12101
org.apache.thrift.transport.TTransportException: backend 10001 shutdown
	at com.starrocks.pseudocluster.PseudoBackend$HeartBeatClient.heartbeat(PseudoBackend.java:707) ~[test-classes/:?]
	at com.starrocks.system.HeartbeatMgr$BackendHeartbeatHandler.call(HeartbeatMgr.java:296) [classes/:?]
	at com.starrocks.system.HeartbeatMgr$BackendHeartbeatHandler.call(HeartbeatMgr.java:274) [classes/:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) [?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
	at java.lang.Thread.run(Thread.java:829) [?:?]
2023-05-17 10:08:01,508 WARN (heartbeat mgr|40) [HeartbeatMgr.runAfterCatalogReady():177] get bad heartbeat response: type: BACKEND, status: BAD, msg: backend 10001 shutdown
2023-05-17 10:08:03,511 WARN (heartbeat-mgr-pool-4|96) [HeartbeatMgr$BackendHeartbeatHandler.call():336] backend heartbeat got exception, addr: 127.0.0.10:12101
org.apache.thrift.transport.TTransportException: backend 10001 shutdown
	at com.starrocks.pseudocluster.PseudoBackend$HeartBeatClient.heartbeat(PseudoBackend.java:707) ~[test-classes/:?]
	at com.starrocks.system.HeartbeatMgr$BackendHeartbeatHandler.call(HeartbeatMgr.java:296) [classes/:?]
	at com.starrocks.system.HeartbeatMgr$BackendHeartbeatHandler.call(HeartbeatMgr.java:274) [classes/:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) [?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
	at java.lang.Thread.run(Thread.java:829) [?:?]
2023-05-17 10:08:03,512 WARN (heartbeat mgr|40) [HeartbeatMgr.runAfterCatalogReady():177] get bad heartbeat response: type: BACKEND, status: BAD, msg: backend 10001 shutdown
